### PR TITLE
Adding Prometheus Exporter to Helm Charts

### PR DESCRIPTION
**Pull request by 0xt3j4s**
## Type
Enhancement

___

## Description
This PR introduces the integration of Prometheus Exporter into the Helm charts of the kubescape-operator. The main changes include:
- Enabling Prometheus Exporter in the Helm charts.
- Adding new YAML configuration files for Prometheus Exporter, including deployment, service, service account, network policy, cluster role, and cluster role binding.
- Updating the values.yaml file to include configurations for Prometheus Exporter.

___

## PR changes walkthrough
<table><thead><tr><th></th><th>Relevant files&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><details><summary>8 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>_helpers.tpl&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/_helpers.tpl<br><br>
        <strong>Added a new configuration for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-aa81f2ba9ca60237f5e62c64faf4e05f73d0f94f6ed0b9773aed5a84aedf993c"> +2/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>clusterrole.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/clusterrole.yaml<br><br>
        <strong>Added a new cluster role for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-364e1a0843b3c591667fda1234f466bbfde41995a38b42a57470cfa3e6bbb209"> +14/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>clusterrolebinding.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/clusterrolebinding.yaml<br><br>
        <strong>Added a new cluster role binding for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-8b7ddc41a537f99d2ff8c3bb7aa5b6f063573c717356324b40fec6d403cfdff0"> +15/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>deployment.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/deployment.yaml<br><br>
        <strong>Added a new deployment configuration for Prometheus <br>Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-f90f59f8051ce40677b2a880bb44e61ca07c515f03990e6144219dd687d1333e"> +82/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>networkpolicy.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/networkpolicy.yaml<br><br>
        <strong>Added a new network policy for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-486d5346bcfcdeb11709bc3dc0936dc86f11ef425496dc22a3742344ba4a421c"> +39/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>service.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/service.yaml<br><br>
        <strong>Added a new service configuration for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-4b8ce7e4189a44a122b4de3ec31723bdf240d82e751aca45dcc5d427d3a0e1f5"> +19/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>serviceaccount.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/serviceaccount.yaml<br><br>
        <strong>Added a new service account for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-f9b699f8e2f9d453bd286f0873eed3b6efa6b0773566d055dc350954ad89843c"> +19/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>values.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/values.yaml<br><br>
        <strong>Updated the values.yaml file to include configurations for <br>Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-2d5611f69251d498d71f52fa778f6ce1bc93e1e1f46951ede1c50d975bdcad02"> +24/-0</a></td>

</tr>                    
</table></details></td></tr></tr></tbody></table>

___

## Type
Enhancement

___
## Description
This PR introduces the integration of Prometheus Exporter into the Helm charts of the kubescape-operator. The main changes include:
- Enabling Prometheus Exporter in the Helm charts.
- Adding new YAML configuration files for Prometheus Exporter, including deployment, service, service account, network policy, cluster role, and cluster role binding.
- Updating the values.yaml file to include configurations for Prometheus Exporter.

___
## PR changes walkthrough
<table><thead><tr><th></th><th>Relevant files&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><details><summary>8 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>_helpers.tpl&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/_helpers.tpl<br><br>
        <strong>Added a new configuration for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-aa81f2ba9ca60237f5e62c64faf4e05f73d0f94f6ed0b9773aed5a84aedf993c"> +2/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>clusterrole.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/clusterrole.yaml<br><br>
        <strong>Added a new ClusterRole for Prometheus Exporter with <br>specific rules.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-364e1a0843b3c591667fda1234f466bbfde41995a38b42a57470cfa3e6bbb209"> +14/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>clusterrolebinding.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/clusterrolebinding.yaml<br><br>
        <strong>Added a new ClusterRoleBinding for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-8b7ddc41a537f99d2ff8c3bb7aa5b6f063573c717356324b40fec6d403cfdff0"> +15/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>deployment.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/deployment.yaml<br><br>
        <strong>Added a new Deployment configuration for Prometheus <br>Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-f90f59f8051ce40677b2a880bb44e61ca07c515f03990e6144219dd687d1333e"> +82/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>networkpolicy.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/networkpolicy.yaml<br><br>
        <strong>Added a new NetworkPolicy for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-486d5346bcfcdeb11709bc3dc0936dc86f11ef425496dc22a3742344ba4a421c"> +39/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>service.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/service.yaml<br><br>
        <strong>Added a new Service configuration for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-4b8ce7e4189a44a122b4de3ec31723bdf240d82e751aca45dcc5d427d3a0e1f5"> +19/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>serviceaccount.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/templates/prometheus-exporter/serviceaccount.yaml<br><br>
        <strong>Added a new ServiceAccount for Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-f9b699f8e2f9d453bd286f0873eed3b6efa6b0773566d055dc350954ad89843c"> +19/-0</a></td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>values.yaml&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </strong></summary>
      <ul>
        charts/kubescape-operator/values.yaml<br><br>
        <strong>Updated the values.yaml file to include configurations for <br>Prometheus Exporter.</strong>
      </ul>
    </details>
  </td>
  <td><a href="https://github.com/kubescape/helm-charts/pull/343/files#diff-2d5611f69251d498d71f52fa778f6ce1bc93e1e1f46951ede1c50d975bdcad02"> +24/-0</a></td>

</tr>                    
</table></details></td></tr></tr></tbody></table>
___
## Overview

- This PR will add the prometheus exporter to the helm charts.

### CODE REVIEW

**Review by dwertent — CHANGES_REQUESTED** [review]
Well done!
Please see my few comments.

In addition, we should add documentation and we should remove the deprecated Prometheus support (currently we support only the kubescape workload).

**Comment by dwertent** [comment]
@0xt3j4s What are the metrics that will be supported? Configuration scanning and image scanning?

File: charts/kubescape-operator/values.yaml [hunk 1]
```diff
@@ -731,3 +732,26 @@ helmReleaseUpgrader:
...
+# Prometheus exporter
+prometheusExporter:
+  name: "prometheus-exporter"
+  enabled: true 
+  image: 
+    repository: "quay.io/kubescape/prometheus-exporter"
+    tag: "latest"
+    pullPolicy: "Always"
+  
+  resources:
```
[the remark this answers is not in the document]
Comment (matthyx) [reply charts/kubescape-operator/values.yaml:746]: no for sure, also we must debug the exporter that's crashing

**Comment by 0xt3j4s** [comment]
Currently, the metrics support configuration scanning (control metrics) and image scanning (vulnerability metrics).

File: charts/kubescape-operator/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xt3j4s) [reply charts/kubescape-operator/values.yaml:746]: Yes, there is some issue with the docker image (keeps crashing).

**Comment by dwertent** [comment]
@0xt3j4s Can you pls fix the conflicts?

File: charts/kubescape-operator/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dwertent) [reply charts/kubescape-operator/values.yaml:754]: @0xt3j4s Was this issue solved?

File: charts/kubescape-operator/templates/prometheus-exporter/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+              mountPath: ~/.kube/config
+              readOnly: true
+      volumes:
+          - name: {{ .Values.global.clusterConfig }}
+            configMap:
+              name: {{ .Values.global.clusterConfig }}
+              items:
+              - key: "config"
+                path: "config"
+
```
Comment (dwertent) [inline charts/kubescape-operator/templates/prometheus-exporter/deployment.yaml:81]: Please add `customScheduling` as introduced [here](https://github.com/kubescape/helm-charts/pull/359/files#diff-a6c71225bcec775a304738cdcfc01ac0feb870555f0fa2297b0aaca4bc62cdd6R56)

File: charts/kubescape-operator/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xt3j4s) [reply charts/kubescape-operator/values.yaml:754]: Not yet; I will have to check with @matthyx once.

[the remark this answers is not in the document]
Comment (matthyx) [reply charts/kubescape-operator/values.yaml:760]: image fixed (using my own quay until we merge https://github.com/kubescape/prometheus-exporter/pull/6)

**Pull request merged**